packages/vscale/src/log-scale.ts

```diff
@@ -156,48 +159,16 @@ export class LogScale extends ContinuousScale {
    */
   stepTicks(step: number): any[] {
     const d = this.calculateVisibleDomain(this._range);
-    return stepTicks(d[0], d[d.length - 1], step);
+    return forceTicksBaseTransform(d[0], d[1], step, this.transformer, this.untransformer);
```

Comment:
don't read `d[1]`, just use `d[d.length - 1]`, in same case ,domain has length > 2